### ISSUE TITLE
If an instance of an Error object is returned in the data then emit that object as an error

### ISSUE THREAD

**Issue by ryanramage** [hughsk/from2#8]
stream errors are tricky. I always forget to test for them. from2 would be super handy if it could emit error if an Error object is introduced in the array. For example

```
from2.obj([
  {a: 1},
  new Error('oooops'),
  {a: 2}
]).on('error', console.log)
 .pipe(ndjson.parse())
```

I can take a crack at this, as I believe I would just add a contains instance of Error in the check function
https://github.com/hughsk/from2/blob/master/index.js#L57

@mafintosh does this sound reasonable?

**Comment by mafintosh** [comment]
@ryanramage i'd add it here instead https://github.com/hughsk/from2/blob/master/index.js#L14 since this is only relevant for the array constructor.

**Comment by ryanramage** [comment]
Fixed and Released as 2.1.0.
